lona/client/dom-renderer.js

```diff
@@ -5,7 +5,9 @@ Lona.LonaDomRenderer = function(lona_context, lona_window) {
     // html rendering ---------------------------------------------------------
     this._render_node = function(node_spec) {
         var _this = this;
-        var property_names = ['value', 'checked'];
+
+        var property_names = ['value', 'disabled', 'checked', 'selected',
+                              'readonly'];
```

Comment:
Why do we need `disabled` and `readonly` in this list?

Reply:
The browser API is very strange in this regard. ``button.disabled`` and ``button.setAttribute('disabled', '')``  both work, for ``checkbox`` and ``checked`` only the first syntax works. I am not sure which syntax makes more here. Maybe we just continue to handle ``value`` and ``checked`` separately.

Reply:
My imagination says that there are two types of properties: "static" and "dynamic" (naming is mine :).

"Static" means that data is stored in html attributes (ex. `disabled`). In another words there is a two way coupling between html attributes and js properties. They are always in sync.

"Dynamic" means that there are two independent data: html attribute and js property (ex. `value`). There is only one type of connection between them: html attribute defines initial value for js property. That's how reset button works for example.

Reply:
Makes sense. But who decided that ``disabled`` is "static" but ``checked`` not!?
Ok, so we keep this list to "dynamic" properties (``value`` and ``checked``) ?

Reply:
I'd say that all properties user can change is "dynamic": `value`, `checked`, `selected`. Others are "static".

Reply:
Ohhhhhh!!!! \*facepalm\*
Ok! Then this list should contain ``value``, ``checked`` and ``selected`` right?

Reply:
In my understanding yes.
But I'm not an expert in HTML and JS :)

Reply:
me neither, but it make sense to me :) Also it works!
I changed it that way.